### DISCUSSION TITLE
NetWorx Desk Band size not adjustable anymore

### DISCUSSION THREAD

**Discussion by Bec-de-Xorbin** [valinet/ExplorerPatcher#1043] [Q&A] [answered]
After updating to 22000.556.42.33, NetWorx Desk Band toolbar size is not adjustable anymore and is stuck at very small size:
![image](https://user-images.githubusercontent.com/1503339/157566815-b40a0f95-5b19-4ff0-b04e-d5a810c044a0.png)

Downgrading to 22000.556.42.32 fixes this:
![image](https://user-images.githubusercontent.com/1503339/157566957-a93be824-1f0e-4a59-9640-7499c9725dc8.png)

Windows version 22000.556.

**Comment by valinet** [comment]
I will take a look at it.

**Comment by valinet** [comment, marked as answer]
Yeah, it was indeed a regression introduced in the previous update. Latest pre-release (42.34) should fix that.
